### ISSUE TITLE
CodeForAustin.org

### ISSUE THREAD

**Issue by mateoclarke** [open-austin/project-ideas#102]
### What problem are we trying to solve?

Code for America gave us a free domain, codeforaustin.org. Right now it redirects to our site. But we could do something more awesome with the domain and make it a nice landing page for more civic tech, code for good opportunities in Austin.

It could link to:
- ATX Hack for Change
- CoA Office of Innovation
- data.austintexas.gov
- transportation.austintexas.io
- https://github.com/open-austin/awesome-austin
- etc...

### Who will benefit (directly and indirectly) from this project?

🤷‍♂️ 

### What help is needed at this time?

- content
- design mockup
- site build (we use Jekyll for a open-austin.org)

----

#### Checklist for NEW ideas :baby:
Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).

- [x] Create this idea issue
- [x] Flesh out the who, where, and what questions above
- [ ] Start the conversation about this new idea on [Slack](https://openaustin-slackin.herokuapp.com/)!
- [ ] Respond with at least one update on this issue within the next month

#### Checklist for ACTIVE projects :fire:
Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:

- [ ] Complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101)
- [ ] Create a [GitHub repository](https://github.com/new) and Slack channel for work.
- [ ] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [ ] [Create issues](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#assignment--add-your-first-issue) to describe each task that you plan to do or need help with and how a contributor can get started on that task. You might start and stop a lot, so consider issues as your to-do list.

#### Checklist for FEATURED Projects :tada:
To have your project FEATURED on [Open-Austin.org](https://open-austin.org/), complete the following documentation. In past projects, well-documented featured projects have more contributions than other projects.

* [ ] In your README, link to the [Open Austin Community Participation Guidelines](https://docs.google.com/document/d/1OujyBccPpepUXSY5_nP3alzvd81WlCKSeSf_-sU5K-U/edit?usp=sharing) or write your own code of conduct.
* [ ] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
* [ ] [Create file: CONTRIBUTING.md](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md) so others know [how they can contribute](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/).

Once all of the above is complete,
- [ ] [Create an issue](https://github.com/open-austin/open-austin.github.io/issues/new) on the open-austin.github.io repo with the title `Add [this project] to projects page. An Open Austin leader will review this issue and post your project :balloon:

If you get stuck at any point, feel free to reach out to the leadership team on Slack by adding @leadership to your message. We're here to help you make real changes to our city.

**Comment by joshavenue** [comment]
@mateoclarke , I am a UI/UX designer introduced by Vickle O'Dell from Open Austin.
My portfolio : http://zensite.design
My email : joshzensite@gmail.com
